src/colmap/sensor/bitmap.cc

```diff
@@ -221,6 +205,41 @@ std::vector<uint8_t> Bitmap::ConvertToColMajorArray() const {
   return array;
 }
 
+std::vector<uint8_t> Bitmap::ConvertToRawBits() const {
```

Comment:
How does this differ to `ConvertToRowMajorArray/ConvertToColMajorArray`?

Reply:
The data may have a pitch that does not equal the width of the image to align the memory.

Reply:
This is how freeimage will internally allocate the memory for faster operations on the image.